modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -285,7 +292,7 @@ def validate_form_526_disability_secondary_disability_required_fields(disability
             raise_exception_if_value_not_present('disabilityActionType',
                                                  "#{form_object_desc}/disabilityActionType")
           end
-          if sd_service_relevance.blank?
+          if sd_disability_action_type == 'NEW' && sd_service_relevance.blank?
```

Comment:
Enum defined in schema for secondary disability action type has one value "SECONDARY", so we'll never hit this. Also couldn't find a validation rule related to service relevance [here](https://confluence.devops.va.gov/pages/viewpage.action?spaceKey=VAExternal&title=Form526+Validation+Rules)? Might want to ask Kayla if the prior check for no service relevance for a secondary disability is correct.